src/app-layer-smtp.h

```diff
@@ -65,6 +65,16 @@ typedef struct SMTPString_ {
     TAILQ_ENTRY(SMTPString_) next;
 } SMTPString;
 
+typedef struct SMTPInput_ {
```

Comment:
would it make sense to separate this out between a `SMTPInput` and a `SMTPLine`?

Reply:
I think so, yes. I have something to share in Rust first.. will send a PR soon